### DISCUSSION TITLE
bug with colors (bg-primary/5) with multiple themes

### DISCUSSION THREAD

**Discussion by gmdaos** [tailwindlabs/tailwindcss#11840] [Help] [answered]
![issue_1](https://github.com/tailwindlabs/tailwindcss/assets/115022551/36e9f349-e662-4af1-a42a-96fd1702b8ef)
![issue_2](https://github.com/tailwindlabs/tailwindcss/assets/115022551/16a52952-6c50-4610-97d6-26007696eb08)
![issue_3](https://github.com/tailwindlabs/tailwindcss/assets/115022551/0ff1a570-f68c-4d52-aca1-9684f6724b05)
![issue_4](https://github.com/tailwindlabs/tailwindcss/assets/115022551/e007404c-9934-4e07-9da5-e205babe0666)

According to the attached images, I have created multiple theme colors including the theme-bon which is applied to an entire page and throughout the page the colors are applied without any error except when you want to give opacity to the background: I am aware of that can be applied in two ways, the first: "bg-primary bg-pacity-5", this code the first is applied however the opacity is no longer applied; and the second method is: "bg-primary/5", with the latter it is applied but the error is that it is not using the theme-bon colors but the default colors.

**Comment by wongjn** [comment]
It could be a bug with what I assume to be some Tailwind plugin you are using for the `themes` key. You could consider reporting a bug to that project.

**Reply by gmdaos** [reply]
Ok I'll do that, and is there a native way to achieve multiple themes with Tailwind?

**Reply by wongjn** [reply, marked as answer]
Yes, you could consider [using CSS variables for color values](https://tailwindcss.com/docs/customizing-colors#using-css-variables).
